engine/server/sv_game.c

```diff
@@ -2393,7 +2394,7 @@ void GAME_EXPORT pfnClientCommand( edict_t* pEdict, char* szFmt, ... )
 		return;
 	}
 
-	if(( client = SV_ClientFromEdict( pEdict, true )) == NULL )
+	if(( client = SV_ClientFromEdict( pEdict, false )) == NULL )
```

Comment:
Seems GoldSource really does not have checks for spawned client. 

Change error message then.